test/browser/features/UnHappyPath/BAVUnhappyAccountDetails.feature

```diff
@@ -11,26 +10,40 @@ Feature: Enter Incorrect Bank Account Details
       When the user clicks the Continue button
       Then an error message is shown to the user
 
-   Scenario Outline:  On-screen error when Account Number in wrong format: <accountNumber>
+   Scenario Outline:  On-screen error when Account Number is the wrong length: <accountNumber>
       Given the user has entered a Sort Code of "123456"
       Given the user has entered an Account Number of <accountNumber>
       When the user clicks the Continue button
-      Then an error message is shown to the user
+      Then an account number length error message is shown to the user
 
       Examples:
          | accountNumber |
-         | "001111111"   |
```

Comment:
Just checking that we haven't removed this because our code no longer passes a valid test scenario?

Reply:
The maxLength limiter is set to 8 so strings any longer than 8 can't be entered in the input field